submissions/ArthurGorbenko/html.css.practice/css/styles.css

```diff
@@ -1,167 +1,176 @@
-body{
-    font-family: arial,sans-serif;
-    font-size: 15px;
+body {
+  font-family: arial, sans-serif;
+  font-size: 15px;
 }
 
-.container{
-    display: flex;
-    justify-content: flex-end;
+.container {
+  display: flex;
+  justify-content: flex-end;
 }
 
-.mail_button, 
+.mail_button,
 .Images {
-    margin: 10px;
+  margin: 10px;
 }
 
-.content{
-    height: 500px;
-    width: 350px;
-    border: 1px solid #ccc;
-    float: right;
-    box-shadow: 0px 0px 10px 0px rgba(0,0,0,0.15);
-    overflow-y: auto;
-    display:none;  
+.content {
+  height: 500px;
+  width: 350px;
+  border: 1px solid #ccc;
+  float: right;
+  box-shadow: 0px 0px 10px 0px rgba(0, 0, 0, 0.15);
+  overflow-y: auto;
+  display: none;
+  position: absolute;
+  margin-top: 50px;
 }
 
 .list_1,
 .list_2 {
-    list-style-type: none;
-    position: relative;
-    padding-inline-start: 20px;
-    margin-block-end:0px;
+  list-style-type: none;
+  position: relative;
+  padding-inline-start: 20px;
+  margin-block-end: 0px;
 }
 
 .card {
-    display: inline-block;
-    height: 98px;
-    width: 86px;
-    padding: 5px;
-    border:1px solid rgb(204, 204, 204, 0);
+  display: inline-block;
+  height: 98px;
+  width: 86px;
+  padding: 5px;
+  border: 1px solid rgb(204, 204, 204, 0);
 }
 
-.card:hover{
-    border:1px solid rgb(204, 204, 204, 0.5);
+
+.card:hover {
+  border: 1px solid rgb(204, 204, 204, 0.5);
 }
 
 .link {
-    text-decoration: none;
-    position: absolute;
-    text-align: center;
-    margin: 10px;
-    position: relative;
-    color: black;
+  text-decoration: none;
+  position: absolute;
+  text-align: center;
+  margin: 10px;
+  position: relative;
+  color: black;
 }
 
 .icon {
-    height: 64px;
-    width: 64px;
-    display: inline-block;
-    vertical-align: top;
+  height: 64px;
+  width: 64px;
+  display: inline-block;
+  vertical-align: top;
 }
 
 .title_btn {
-    text-align: center;
-    display: block;
-    width: 84px;
+  text-align: center;
+  display: block;
+  width: 84px;
+}
+input {
+  position: absolute;
+  overflow-y: auto;
+  clip: rect(0 0 0 0);
 }
 
-#trig,
-#more {
-    display: none;
+#trig:focus + label {
```

Comment:
Avoid styling by #id, it's harder to override in the future.

Reply:
[Fixed](https://github.com/kottans/frontend-2019-p2p/pull/67/commits/0f313d0350645b3670fb94ebc61e074b6a38e544)